app/src/main/java/net/gsantner/opoc/util/FileUtils.java

```diff
@@ -409,9 +411,13 @@ public static String getMimeType(File file) {
 
             if (guess == null || guess.isEmpty()) {
                 guess = "*/*";
-                int dot = file.getName().lastIndexOf(".") + 1;
-                if (dot > 0 && dot < file.getName().length()) {
-                    switch (file.getName().substring(dot)) {
+                String filename = file.getName();
+                if (filename.endsWith(Constants.ENCRYPTION_EXTENSION)) {
+                    filename = filename.substring(0, filename.length() - Constants.ENCRYPTION_EXTENSION.length());
+                }
+                int dot = filename.lastIndexOf(".") + 1;
+                if (dot > 0 && dot < filename.length()) {
+                    switch (filename.substring(dot)) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A switch statement does not contain a break](https://app.codacy.com/manual/gsantner/markor/pullRequest?prid=5279643)